lib/utils.c

```diff
@@ -666,3 +667,9 @@ pqos_mba_ctrl_enabled(const struct pqos_cap *cap,
 
         return PQOS_RETVAL_OK;
 }
+
+enum pqos_vendor
+pqos_get_vendor(void)
```

Comment:
```suggestion
pqos_get_vendor(const struct pqos_cpuinfo *cpu)
```
Or you can use _pqos_cap_get to obtain cpuinfo structure

Reply:
Michal, Any concerns about this pull request?  Are you waiting for any more changes?